docs/se/reactivestreams/02_engine.adoc

```diff
@@ -0,0 +1,153 @@
+|skip|Skip first n items, all the others are emitted.
+|flatMap|Transform item with supplied function and flatten resulting `Flow.Publisher` to downstream.
+|flatMap|Transform item with supplied function and flatten resulting `Flow.Publisher` to downstream while limiting the maximum number of concurrent inner `Flow.Publisher`s and their in-flight item count, optionally aggregating and delaying all errors until all sources terminate.
+|flatMapIterable|Transform item with supplied function and flatten resulting `Iterable` to downstream.
+|observeOn|Re-emit the upstream's signals to the downstream on the given executor's thread using a default buffer size of 32 and errors skipping ahead of items.
+|observeOn|Re-emit the upstream's signals to the downstream on the given executor's thread.
+|flatMapIterable|Transform item with supplied function and flatten resulting `Iterable` to downstream.
+|forEach|Terminal stage, invokes provided consumer for every item in the stream.
+|collectList|Collect the items of this `Multi` instance into a `Single` of `List`.
+|collect|Collect the items of this `Multi` instance into a `Single`.
+|collect|Collect the items of this `Multi` into a collection provided via a `Supplier`
+*and mutated by a {@code BiConsumer} callback.
+|collectStream|Collects up upstream items with the help of a the callbacks of a `java.util.stream.Collector`.
+|reduce|Combine subsequent items via a callback function and emit the final value result as a Single.
+|reduce|Combine every upstream item with an accumulator value to produce a new accumulator value and emit the final accumulator value as a Single.
+|first|Get the first item of this `Multi` instance as a `Single`.
+|from|Wrap a CompletionStage into a Multi and signal its outcome non-blockingly.
+|from|Wrap a CompletionStage into a Multi and signal its outcome non-blockingly.
+|from|Create a `Multi` instance wrapped around the given publisher.
+|from|Create a `Multi` instance that publishes the given iterable.
+|from|Create a `Multi` instance that publishes the given `Stream`.
+|just|Create a `Multi` instance that publishes the given items to a single subscriber.
+|just|Create a `Multi` instance that publishes the given items to a single subscriber.
+|singleton|Create a `Multi` that emits a pre-existing item and then completes.
+|error|Create a `Multi` instance that reports the given exception to its subscriber(s). The exception is reported by invoking `Subscriber#onError(java.lang.Throwable)` when `Publisher#subscribe(Subscriber)` is called.
+|empty|Get a `Multi` instance that completes immediately.
+|never|Get a `Multi` instance that never completes.
+|concat|Concat streams to one.
+|onTerminate|Executes given `java.lang.Runnable` when any of signals onComplete, onCancel or onError is received.
+|onComplete|Executes given `java.lang.Runnable` when onComplete signal is received.
+|onError|Executes given `java.lang.Runnable` when onError signal is received.
```

Comment:
Executes the given `java.util.function.Consumer` when an `onError` signal is received.

Reply:
thx